### PR TITLE
WIP Anime guide

### PR DESCRIPTION
**Pull request by FonduemangVI**
# Pull request

**Purpose**
Pushing first draft of Anime Guide with some additonal changes

**Approach**
If this Pull Request is created to solve a issue explain how does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
If you're adding a new Custom Format make sure you follow the [Radarr/Sonarr Custom Format (JSON) Guidelines](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/sonarr/cf/uncensored.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "026d5aadd1a6b4e550b134cb6c72b3ca",
+  "trash_score": "0",
```
Comment (TRaSH-) [inline docs/json/sonarr/cf/uncensored.json:3]: To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

File: docs/json/sonarr/cf/anime-dual-audio.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "418f50b10f1907201b6cfdf881f467b7",
+  "trash_score": "0",
```
Comment (TRaSH-) [inline docs/json/sonarr/cf/anime-dual-audio.json:3]: To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

File: docs/json/sonarr/cf/10bit.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "b2550eb333d27b75833e25b8c2557b38",
+  "trash_score": "0",
```
Comment (TRaSH-) [inline docs/json/sonarr/cf/10bit.json:3]: To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

File: docs/json/sonarr/cf/uncensored.json (hunk 1 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/uncensored.json]: Spoke with Nit, you are correct have to remove the 0 scores

File: docs/json/sonarr/cf/anime-dual-audio.json (hunk 2 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/anime-dual-audio.json]: Spoke with Nit, you are correct have to remove the 0 scores

File: docs/json/sonarr/cf/10bit.json (hunk 3 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/10bit.json]: Spoke with Nit, you are correct have to remove the 0 scores

File: docs/Sonarr/Sonarr-setup-custom-formats-Anime.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+# How to setup Custom Formats (Anime)
+
+!!! attention
+    You must be running Sonarr V4 to be able to use this setup
+
+!!! note
+    This guide is created and maintained by "Update Later"
```
Comment (TRaSH-) [inline docs/Sonarr/Sonarr-setup-custom-formats-Anime.md:7]: Update later ?

Comment (FonduemangVI) [reply docs/Sonarr/Sonarr-setup-custom-formats-Anime.md:7]: Placeholder haha hadn't filled it out yet

**Pull request closed without merging**